tdvt/tdvt/exprtests/regexcalcs/expected.setup.string.regex.icu_fallback.2.txt

```diff
@@ -0,0 +1,909 @@
+<results>
+  <test name='REGEXP_EXTRACT(&apos;http://academic.research.microsoft.com/Author/232694&apos;, &apos;((https|http)+:\/\/+(www\.)?+[a-z0-9\.:-]+\/)&apos;)'>
+    <sql>
+      SELECT TABLEAU.REGEXP_EXTRACT(&apos;http://academic.research.microsoft.com/Author/232694&apos;, E&apos;((https|http)+:\\/\\/+(www\\.)?+[a-z0-9\\.:-]+\\/)&apos;) AS &quot;TEMP(Test)(368590675)(0)&quot;&#10;FROM &quot;Extract&quot;.&quot;Extract&quot; &quot;Extract&quot;&#10;HAVING (COUNT(1) &gt; 0)&#10;    </sql>
+    <query-time>
+      5.000&#10;    </query-time>
```

Comment:
sql and query-time elements are suppressed in the other expected files

Reply:
TDVT ignores those sections when doing the comparison, but I will make sure they are removed to keep things tidy.

Reply:
Removed in 783f54290694aec4f079b74d08009d1e79203ab9